stream-chat-android-client/src/main/java/io/getstream/chat/android/client/channel/ChannelClient.kt

```diff
@@ -690,7 +690,7 @@ public class ChannelClient internal constructor(
         sort: QuerySort<Member> = QuerySort(),
         members: List<Member> = emptyList(),
     ): Call<List<Member>> {
-        return client.queryMembers(channelType, channelId, offset, limit, filter, sort, members)
+        return client.queryMembersInternal(channelType, channelId, offset, limit, filter, sort, members)
```

Comment:
Why are we skipping side effects here? Calling methods from `ChannelClient` should have the same behavior as those from `ChatClient`